services/provider/server/server.go

```diff
@@ -356,7 +356,7 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 					Data: mustMarshal(map[string]string{
 						"fsid":          fsid,
 						"mon-secret":    "mon-secret",
-						"ceph-username": fmt.Sprintf("client.cephclient-health-checker-%s", consumerResource.Name),
+						"ceph-username": fmt.Sprintf("client.%s", i.Name),
```

Comment:
This i is confusing, I would also suggest that you change the name of the i to something else as i is mainly used for the index which is not the case here.

Reply:
changed to `cephRes`